### PR TITLE
Jandm gwtii 292 wfs string

### PR DESCRIPTION
**Pull request by jandm**
GWTII-292: CriterionBuilder missing type for string attribute
GWTII-293: WFS server strategy support

### CODE REVIEW

File: plugin/wfs/server-extension/src/test/java/org/geomajas/gwt2/plugin/wfs/client/query/CriterionDtoBuilderTest.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.geomajas.gwt2.plugin.wfs.client.query;
+
+import org.junit.Test;
+
+public class CriterionDtoBuilderTest {
+
```
Comment (janvenstermans) [inline plugin/wfs/server-extension/src/test/java/org/geomajas/gwt2/plugin/wfs/client/query/CriterionDtoBuilderTest.java]: Test file that can be removed.

**Comment by janvenstermans** [comment]
Jan Venstermans: +1

**Comment by oliverm-be** [comment]
+1 (binding)

**Comment by pieterdg** [comment]
+1 (binding)

**Pull request merged**